src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -232,3 +236,47 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
         cache.delete(cache_key)
         metrics.incr("github_pr_comment.api_error")
         raise e
+
+
+@instrumented_task(name="sentry.tasks.integrations.github_comment_reactions")
+def github_comment_reactions():
+    logger.info("github.pr_comment.reactions_task")
+
+    comments = PullRequestComment.objects.filter(
```

Comment:
you can further optimize this by retrieving the client only once per integration. I'm fine leaving it for now but keeping track of task run time.

Reply:
i wrote something to try to optimize it, but i feel like it's gonna OOM